evm_loader/program/src/external_programs/metaplex.rs

```diff
@@ -90,16 +90,7 @@ fn create_metadata_accounts_v3(
 
     assert_collection_update_is_valid(false, &None, &args.data.collection)?;
     metadata.collection = args.data.collection.clone();
-
-    if let Some(details) = &args.collection_details {
-        match details {
-            CollectionDetails::V1 { size: _size } => {
-                metadata.collection_details = Some(CollectionDetails::V1 { size: 0 });
-            }
-        }
-    } else {
-        metadata.collection_details = None;
-    }
+    metadata.collection_details = None;
```

Comment:
Removed because of `Use of deprecated variant` warning:
```rust
#[repr(C)]
#[cfg_attr(feature = "serde-feature", derive(Serialize, Deserialize))]
#[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
pub enum CollectionDetails {
    #[deprecated(
        since = "1.13.1",
        note = "The collection size tracking feature is deprecated and will soon be removed."
    )]
    V1 { size: u64 },
}
```